sunpy/spectra/spectrogram.py

```diff
@@ -274,28 +274,28 @@ class Spectrogram(Parent):
 
     Attributes
     ----------
-    data : `~numpy.ndarray`
```

Comment:
The ~ will be needed here. It hides certain parts when the documentation is built.